laws/src/main/scala/cats/laws/discipline/Arbitrary.scala

```diff
@@ -165,6 +166,22 @@ object arbitrary extends ArbitraryInstances0 {
 
   implicit def catsLawsArbitraryForReaderWriterStateT[F[_]: Applicative, E, L, S, A](implicit F: Arbitrary[(E, S) => F[(L, S, A)]]): Arbitrary[ReaderWriterStateT[F, E, L, S, A]] =
     Arbitrary(F.arbitrary.map(ReaderWriterStateT(_)))
+
+  implicit def catsLawsArbitraryForListNatTrans: Arbitrary[List ~> List] =
```

Comment:
These two arbitrary instances seems a bit too, well, arbitrary and limited to be included in discipline for general uses. (actually the second one might be unused according to the codcov), how about we move them into the tests?